src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -243,4 +252,16 @@ protected function usingCookieSessions()
 
         return $this->manager->driver()->getHandler() instanceof CookieSessionHandler;
     }
+
+    /**
+     * Determines if the specified session needs garbage collection for expired
+     * sessions.
+     *
+     * @param  \Illuminate\Session\SessionInterface  $session
+     * @return bool
+     */
+    protected function gcNeeded(SessionInterface $session)
+    {
+        return ! ($session->getHandler() instanceof CacheBasedSessionHandler);
```

Comment:
> Imo the length of the expression followed by the ! negation justifies the brackets because it makes the conditional statement more readable and expressive.

They are unnecessary. At the end of the day it doesn't matter though. I'll just remove them myself after merging so this matches our code style.